gramps/plugins/importer/importxml.py

```diff
@@ -279,21 +278,21 @@ def info_text(self):
         Construct an info message from the data in the class.
         """
         key2string = {
-            PERSON_KEY      : _('  People: %d\n'),
-            FAMILY_KEY      : _('  Families: %d\n'),
-            SOURCE_KEY      : _('  Sources: %d\n'),
-            EVENT_KEY       : _('  Events: %d\n'),
-            MEDIA_KEY       : _('  Media Objects: %d\n'),
-            PLACE_KEY       : _('  Places: %d\n'),
-            REPOSITORY_KEY  : _('  Repositories: %d\n'),
-            NOTE_KEY        : _('  Notes: %d\n'),
-            TAG_KEY         : _('  Tags: %d\n'),
-            CITATION_KEY    : _('  Citations: %d\n'),
+            PERSON_KEY      : _('\tPeople{colon}\t%d\n').format(colon=COLON),
+            FAMILY_KEY      : _('\tFamilies{colon}\t%d\n').format(colon=COLON),
+            SOURCE_KEY      : _('\tSources{colon}\t%d\n').format(colon=COLON),
+            EVENT_KEY       : _('\tEvents{colon}\t%d\n').format(colon=COLON),
+            MEDIA_KEY       : _('\tMedia Objects{colon}\t%d\n').format(colon=COLON),
+            PLACE_KEY       : _('\tPlaces{colon}\t%d\n').format(colon=COLON),
+            REPOSITORY_KEY  : _('\tRepositories{colon}\t%d\n').format(colon=COLON),
+            NOTE_KEY        : _('\tNotes{colon}\t%d\n').format(colon=COLON),
+            TAG_KEY         : _('\tTags{colon}\t%d\n').format(colon=COLON),
+            CITATION_KEY    : _('\tCitations{colon}\t%d\n').format(colon=COLON),
             }
         txt = _("Number of new objects imported:\n")
         for key in self.keyorder:
             if any(self.data_unknownobject):
-                strng = key2string[key][0:-1] + ' (%d)\n'
+                strng = key2string[key][0:-1] + '\n(%d)\n'
```

Comment:
Why do you want the unknown object count on a new line?

There is also a problem for RTL languages with using string concatenation here.  I'll fix it in a separate PR.

Reply:
Oh, it's my error. Sorry...